SETUP_ARCH.md

```diff
@@ -0,0 +1,23 @@
+# Snowdrift setup instructions for Arch Linux
+
+Arch has the benefit of being very well-documented, so this document is
+very short!
+
+1.  [Install Haskell][1]. While you are at it, make sure you have the
+    `base-devel` group installed.
+2.  [Install PostgreSQL][2]. You don't need to go through setting up a
+    user or anything like that. Snowdrift will do that for you.
```

Comment:
"or anything like that" ? I think I would say "Note: you don't need to set up a postgres user at this time" and leave it at that, or something of that ilk. And don't say "Snowdrift will dot that" because sdm isn't Snowdrift, it could confuse people as to how much Snowdrift messes with their system.